prez/content/a10.md

```diff
@@ -3,31 +3,25 @@
 >Occur whenever a web application is fetching a remote resource without validating the user-supplied URL.<br><br>This allows an attacker to force the application to send a crafted request to an unexpected destination, even when protected by a firewall, VPN etc..s
 
 ---
-## Notable Common Weakness Enumerations (CWEs)
 
-Notable Common Weakness Enumerations (CWEs):
+## Notable Common Weakness Enumerations (CWEs)
 
 - [CWE-918:](https://cwe.mitre.org/data/definitions/918.html)
 Server-Side Request Forgery (SSRF)
 
 ---
-## Your Tasks 🧑🏻‍💻
-
-- Explore related CWEs (918)/CVEs
-  - `https://cwe.mitre.org/` - `https://www.opencve.io/`<!-- .element: style="font-size:0.8em"-->
-  - e.g. 918 - `https://www.opencve.io/cve?cwe=CWE-918`<!-- .element: style="font-size:0.8em"-->
 
+## Your Tasks 🧑🏻‍💻
 
 - Solve challenges in `juice-shop`
```

Comment:
The task is not related to the heading - SSRF. 
May be add a disclaimer that the SSRF task in juiceshop is not beginner friendly and that is why you would demonstrate the example instead of them doing it ? Up to you how want to do it but if the workshop is about OWASP Top 10 then the participants would probably be expecting to gain an understanding about all of the top 10 vulnerabilities.

Reply:
Yeah, good point. I remember this feedback now that you mention it. 
I really like going through the examples given in that task as they are quite optical, but yes, they are not really SSRF. I think this needs to be solved if we decide to change the workshop content. I'll write it down as a improvement point as this task is not a new task in this PR.